Gurux.DLMS.python/gurux_dlms/objects/GXDLMSExtendedRegister.py

```diff
@@ -169,7 +170,9 @@ def setValue(self, settings, e):
                     if settings.isServer:
                         self.value = e.value
                     else:
-                        self.value = e.value * self.scaler
+                        decimals = math.log10(self.scaler)
```

Comment:
This will never work since round/2 must have an integer.

Reply:
Hi,
Scaler is changed to float when it's read from the meter. Check this from few lines below.

BR,
Mikko

Reply:
Hi,

This is verified with Python 3.7.4. Register data is integer value 2345 and the scaler is 0.001. The value is shown as 2.345. What version of Python you are using? We can fix this after this we can repeat this.

BR,
Mikko